graviti/portex/factory.py

```diff
@@ -89,12 +192,24 @@ def __call__(self, **kwargs: Any) -> PortexType:
         Returns:
             The applied Portex type.
 
+        """
+        return self._class(**self.get_kwargs(**kwargs))  # type: ignore[call-arg]
+
+    def get_kwargs(self, **kwargs: Any) -> Dict[str, Any]:
```

Comment:
will this method be called more than once in a instantiation process. can we refine it?
also do we have a better name for it?